go/vt/sqlparser/ast.go

```diff
@@ -473,11 +473,8 @@ func (*ValuesStatement) iSelectStatement() {}
 
 // Select represents a SELECT statement.
 type Select struct {
-	Cache         string
-	CalcFoundRows bool
 	Comments      Comments
-	Distinct      string
-	Hints         string
+	QueryOpts     []string
```

Comment:
We don't need each option as a separate field in the Select struct, but a string slice is too free form. You probably want to have a QueryOpts struct to hold these things during list construction with a `.Merge()` method like we have with column type options